### ISSUE TITLE
Unified Formatting

### ISSUE THREAD

**Issue by TheCaptainCraken** [themarcman314/ftwgPassMan#14]
We could use some type of unified formatting with a shared configuration file. Also we could settle on some naming conventions etc. This way the number of changes in every committing would actually reflect the actual change in the code.

**Comment by themarcman314** [comment]
Yep, vieni su discord Pietro?

**Comment by themarcman314** [comment]
So creating new c and h files to organise the project and having a unified config parameters?

**Comment by TheCaptainCraken** [comment]
No, something like [this](https://astyle.sourceforge.net/). Basically you use a program that formats the code for you. The only condition is that the program is configurable so that everyone can use the same config. (It's usually done with a file in the root of the repo)
